plugins/woo-ai/includes/class-woo-ai-settings.php

```diff
@@ -0,0 +1,181 @@
+			array(
+				'id'    => 'ai_describe_store',
+				'type'  => 'title',
+				'title' => __( 'About your store', 'woocommerce' ),
+				'desc'  => __( "Tell us a bit more about your business, and our AI will get to work on generating your content, images, and look and feel of your store. And if it's not quite right you can always customize it later.", 'woocommerce' ),
+			),
+
+			array(
+				'id'    => 'ai_describe_store_sell',
+				'type'  => 'text',
+				'title' => __( 'What does your store sell?', 'woocommerce' ),
+				'desc'  => __( 'e.g. organic fruit and vegetables', 'woocommerce' ),
+				'css'   => 'min-width:300px;',
+			),
+
+			array(
+				'id'          => 'ai_describe_store_description',
+				'type'        => 'textarea',
+				'title'       => __( 'Describe your store', 'woocommerce' ),
+				'desc_tip'    => __( 'The more detail you provide, the better job our AI can do! Try to include what makes your business unique, who your audience is, and how many products you plan to display.', 'woocommerce' ),
+				'placeholder' => __( 'e.g. Marianne Renoir is a greengrocery taken over by a ten generations Parisian family who wants to keep quality and tradition in the quarter of Montmartre', 'woocommerce' ),
+				'css'         => 'min-width:300px;min-height: 130px;',
+			),
+
+			array(
+				'type' => 'sectionend',
+				'id'   => 'ai_describe_store',
+			),
+
+			array(
+				'title' => __( 'Tone of voice', 'woocommerce' ),
+				'type'  => 'title',
+				'desc'  => __( "Choose the tone of voice you'd like to use to communicate with your customers.", 'woocommerce' ),
+				'id'    => 'ai_tone_of_voice',
+			),
+
+			array(
+				'title'    => __( 'Tone', 'woocommerce' ),
+				'id'       => 'ai_tone_of_voice_select',
+				'class'    => 'wc-enhanced-select',
+				'css'      => 'min-width:300px;',
+				'default'  => 'informal',
+				'type'     => 'select',
+				'options'  => array(
+					'informal'     => __( 'Informal', 'woocommerce' ),
+					'humorous'     => __( 'Humorous', 'woocommerce' ),
+					'neutral'      => __( 'Neutral', 'woocommerce' ),
+					'youthful'     => __( 'Youthful', 'woocommerce' ),
+					'formal'       => __( 'Formal', 'woocommerce' ),
+					'motivational' => __( 'Motivational', 'woocommerce' ),
+				),
+				'desc_tip' => true,
+			),
+
+			array(
+				'type' => 'sectionend',
+				'id'   => 'ai_tone_of_voice',
+			),
+
+			array(
+				'title' => __( 'Store style', 'woocommerce' ),
+				'type'  => 'title',
+				'desc'  => __( "Select the look and feel that best fits your brand - we'll base the layout, color palette, and fonts used on your store around this.", 'woocommerce' ),
+				'id'    => 'ai_store_style',
+			),
+
+			array(
+				'title'    => __( 'Style', 'woocommerce' ),
+				'id'       => 'ai_store_style_select',
+				'class'    => 'wc-enhanced-select',
+				'css'      => 'min-width:300px;',
+				'default'  => 'aa',
+				'type'     => 'select',
+				'options'  => array(
+					'modern'    => __( 'Modern', 'woocommerce' ),
```

Comment:
I think it's technically possible for the translation strings to get hacked so it might be a good idea to escape these strings rather than directly output them? It might be overkill but WC core does it:
https://woocommerce.github.io/code-reference/files/woocommerce-includes-admin-class-wc-admin-pointers.html

Reply:
Good call 👍